app/helpers/declarable_helper.rb

```diff
@@ -1,4 +1,14 @@
 module DeclarableHelper
+  def classification_description(commodity)
```

Comment:
We should add a test for this guy :simple_smile:

Reply:
👍